websub-ballerina/sub_listener.bal

```diff
@@ -320,10 +320,7 @@ isolated function initiateSubscription(SubscriberServiceConfiguration serviceCon
     SubscriptionClient subscriberClientEp = check getSubscriberClient(hubUrl, serviceConfig?.httpConfig);
     SubscriptionChangeRequest request = retrieveSubscriptionRequest(topicUrl, callbackUrl, serviceConfig);
     var response = subscriberClientEp->subscribe(request);
-    if response is SubscriptionChangeResponse {
-        string subscriptionSuccessMsg = string`Subscription Request successfully sent to Hub[${response.hub}], for Topic[${response.topic}], with Callback [${callbackUrl}]`;
-        log:printInfo(string`${subscriptionSuccessMsg}. Awaiting intent verification.`);
```

Comment:
Rather removing them, can we change the log level into `debug`. Because not there is not trace for the `else` case and can be useful if an issue occurs